### DISCUSSION TITLE
Coinflip automation

### DISCUSSION THREAD

**Discussion by Maxhem2** [thejoabo/virtualfisher-bot#88] [Ideas]
Basically it would be nice if there was an option to risk coinflipping. Always bet on heads, and if lost double the investment next time to make it even. Ofcourse this would requiere much capital but maybe there can be a percentage from the total that will be used to invest securely and make sure the coinflip can even fail 20 times

**Comment by thejoabo** [comment]
Hello @Maxhem2, that's an interesting idea. I'm willing to work on this idea with a statistical approach, to actually use a valid strategy for "safe" gambling. But I need help with the mathematical side of things. In my experience, brute-forcing things - especially in this case, that manipulate large amounts of capital - is a way to failure. 

I'll research some known methods of this type of gambling and give it a try.

Thanks for your suggestion
